odd-platform-ui/src/components/shared/TruncatedCell/TruncatedCellMenu/TruncatedCellMenu.tsx

```diff
@@ -40,7 +41,7 @@ const TruncatedCellMenu: React.FC<TruncatedCellMenuProps> = ({
         onClick={handleMenuOpen}
         height={20}
       />
-      <S.ListMenu
+      <AppMenu
```

Comment:
You forget to pass maxWidth and maxHeight props to this component. These values was in deleted styled-component (ListMenu).